pkg/plugin/processor/builtin/registry.go

```diff
@@ -41,6 +46,10 @@ type blueprint struct {
 
 type ProcessorPluginConstructor func(log.CtxLogger) sdk.Processor
 
+func NewDefaultRegistry(logger log.CtxLogger) *Registry {
+	return NewRegistry(logger, defaultBuiltinProcessors)
+}
```

Comment:
tbh I don't see a real need for this function, on the contrary, I'd rather see a call to `NewRegistry` and know what constructors were provided explicitly. I would rather ...